azurepipelines/build/native/adu-ubuntu-amd64-build.yml

```diff
@@ -83,28 +83,28 @@ extends:
     stages:
       - stage: Build
         jobs:
-          - job: BuildAduAgent_ubuntu2204
-            displayName: "Build ADU Agent - Ubuntu 22.04 (amd64)"
+          - job: BuildAduAgent_ubuntu2004
+            displayName: "Build ADU Agent - Ubuntu 20.04 (amd64)"
             timeoutInMinutes: 120
```

Comment:
Should we consider reducing the build timeout? The total build time should be less than 60 minutes (or 40 minutes).
Or, may be we should set timeout per job, inside the adu-native-build-steps.yml, if possible.

Reply:
Let's set it to 30 mins. The idea is if it takes longer than that, it can be considered a "failed build", possibly due to ADO infra issues OR it gives us incentive to make the builds as fast as possible, which is a good thing.

Reply:
This is a good bug that should be related to larger items not just the ubuntu build pipelines.